### DISCUSSION TITLE
Yarn build works normal but when deploying i get an error ?

### DISCUSSION THREAD

**Discussion by sadiki-o** [vercel/community#250] [Help] [answered]
Hey guys i am trying to deploy my first nextjs project to vercel, i build it in my local machine and it works amazingly fast and normal but when  deploy it using the browser it takes lot and at certain point it freezes the browser and system, that is when i tried to deploy it using the CLI but yes here is the error i get which i have no idea what is causing it:

since the log is so long i put it into a txt [file](https://gofile.io/d/FwBmVe)

**Comment by leerob** [comment, marked as answer]
Based on the error provided, that doesn't seem like a Vercel specific issue - could you share an example of the build working locally? What type of application is this?

**Updated Answer: using yarn, instead of nam, resolved the conflicts**

**Reply by sadiki-o** [reply]
Hey thank u very much for the reply, i know u are very busy with work and answering people here and there "much appreciated", so i was able to actually figure out the issue , what i was trying to build is a fullstack app using supabase auth and postgres database including some ui libraries such as chakra ui and emotion, but the issue was npm ... it truly causes lot of issues especially when installing packages and resolving conflicts, what i did was creating a new nextjs project and installing all the packages i had with the old one through yarn next i moved the pages, components and utils folders to the newly created project and run vercel on terminal and tadaa my project got deployed successfully . 
generally speaking npm is my go through package manager but yes it certainly causes lot of issues in the last months that why i recommend yarn in order to avoid issues during deployement.
